bindings/wasm/examples/src/create_vc.js

```diff
@@ -38,8 +38,8 @@ async function createVC(clientConfig) {
     // Sign the credential with the Issuer's newKey
     const signedVc = issuer.doc.signCredential(unsignedVc, {
         method: issuer.doc.id().toString() + "#newKey",
-        public: issuer.newKey.public(),
-        private: issuer.newKey.private(),
+        public: Array.from(issuer.newKey.public()),
+        private: Array.from(issuer.newKey.private()),
     }, SignatureOptions.default());
```

Comment:
I feel like this is an unacceptable loss of ergonomics for developers. What type does `Array.from` return? What if developers forget to call `Array.from`, is it type-enforced? Is it just a problem with the duck-typed interface here or in general? Can we correct it if it's the former?

Ideally they should just have to call `.public()` or `.private()`. We may have to reconsider `UInt8Array` and instead use key wrapper types like `Public`/`Private` that can be converted to `UInt8Array` as-needed to avoid situations like this.

If we are forced to use `Array.from(uint8array)` or `[uint8array.buffer]` that's fine, but I'd like to have all options explored before that.

Reply:
If that's the only location causing problems (and we'll have to double-check that it is), then making `Args` a proper duck-typed interface with structural getters rather than going through serde would be the solution.

Reply:
As we discussed, since #755  removed this, it's not needed.